specs/mgt-picker.md

```diff
@@ -35,12 +36,31 @@ The picker is a component that can query any Microsoft Graph API and render a dr
 
 `<mgt-picker resource="/groups" version="beta" scopes="Group.Read.All" cache-enabled="true" cache-invalidation-period="50000"></mgt-picker>`
 
+### Advanced Examples
+
+```
+<mgt-picker resource="/groups">
+  <template type="rendered-item">
+    Item {{id}} with title {{title}}
+  </template>
+</mgt-picker>
+
+<mgt-picker resource="/groups">
+  <template type="rendered-item">
+    <span>
+      <mgt-person person-details="{{ group }}" person-card="none" fetch-image></mgt-person> - {{title}}
```

Comment:
The rendered item template shouldn't be aware of the resource being used and should be pretty generic. as such the `person-details="{{ group }}" ` attribute here doesn't makes sense to me. Shouldn't this be `{{ item }}` instead?

Reply:
It absolutely should be `{{ item }}`. Good catch!